pkg/referrerstore/oras/oras.go

```diff
@@ -269,6 +281,7 @@ func (store *orasStore) GetBlobContent(ctx context.Context, subjectReference com
 	}
 
```

Comment:
nit: probably we could just have one line: metrics.ReportBlobCacheCount(ctx, isCached)